src/app/Interface/country.ts

```diff
@@ -0,0 +1,4 @@
+export interface CountryI {
```

Comment:
Should we add this as a convention or should we get rid of the I for interfaces?

Reply:
changed naming to ICountry to match other interfaces naming